docs/reference/pip_install.rst

```diff
@@ -16,14 +16,17 @@ Description
 .. pip-command-description:: install
 
 
-.. _`Requirements File Format`:
-
 Installation Order
 ++++++++++++++++++
 
-Pip installs dependencies before the things that depend on them. In the event
-of a dependency cycle, the first encountered member of the cycle is installed
-last.
+As of v6.1.0, pip installs dependencies before their dependents, i.e. in
```

Comment:
Side note, I'm thinking maybe we should do this as a v7.0 if @rbtcollins gets either the wheel building or setup requires PR landed just because of the chance for breakage. Not really asking for this to get changed though, just pointing it out.